expyfun/stimuli/_tracker.py

```diff
@@ -239,6 +248,22 @@ def respond(self, correct):
                 if self._direction <= 0:
                     self._direction = 1
 
+        if self._repeat_limit == 'reversals':
+            if self._x[-1] == self._x_min:
```

Comment:
@rkmaddox and I were discussing this a few weeks ago and I think our consensus was that having a reversal without a change in direction was a little odd mathematically. @rkmaddox is that still your feeling on the issue?